### ISSUE TITLE
Issue with Ubidots API

### ISSUE THREAD

**Issue by ranasaurus9** [universam1/iSpindel#571]
I am trying to configure my ispindel with Ubidots API however after supplying the API token on the config page and successful reboot, its still not showing up in my device list.

Any idea what's wrong?

## Your Environment

- Firmware Version used: 7.1.4
- Browser Name and version: Google chrome (v98)
- Service backend you have configured the iSpindel to talk to: Ubidots

**Comment by PharmaJap** [comment]
Make sure you're using the "default token", and not the API key or app token. Also verify that the iSpindel is actually connecting to your WiFi (check from the router, or by using another service). Hard to guess without more info.

**Comment by chris160562** [comment]
I encountered exactly the same problems - connects to my WiFi, but does not want to send the info to Ubidots - made sure I am using the default Token. Seems to be a problem resolving the DNS of "industrial.api.ubidots.com". I substituted the ip-address of "industrial.api.ubidots.com" into the UBISERVER variable recompiled it and it worked. This is just a temp work around, and not permanent fix.

An idea why the DNS would not be resolved?

Using firmware 7.2.1 in windows 11 environment.
